js/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-- Node.JS 4,5, or 6+
```

Comment:
True enough, I guess.  If you want to compare the performance between Node versions.